Web/starter_project_barbies/components/Home-page/ImpactStories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Image from 'next/image'
+import impactStroyOne from "../../public/img/home-page/impact-stories/impact-story-1.png"
+import impactStroyTwo from "../../public/img/home-page/impact-stories/impact-story-2.png"
+import impactStroyThree from "../../public/img/home-page/impact-stories/impact-story-3.png"
+import impactStroyFour from "../../public/img/home-page/impact-stories/impact-story-4.png"
+import impactStroyFive from "../../public/img/home-page/impact-stories/impact-story-5.png"
+import impactStroySix from "../../public/img/home-page/impact-stories/impact-story-6.png"
+import impactStroySeven from "../../public/img/home-page/impact-stories/impact-story-7.png"
+function ImpactStories() {
```

Comment:
i would rather have this loaded as data entry instead of being hard coded in the component

Reply:
you can assume the backend has a list of impact stories stored somewhere (you can use a JSON file inside data folder), and you load them and use it as an ImpactStory data type (which you might need to define in the types)

Then you could render in the component by looping over the list of impact stories

Reply:
this probably means that most of the image url's would be inside of that data json and you would not need to load it one by one here

you can also assume that data and that link could change without having to modify the code